app/uk/gov/hmrc/cbcr/repositories/FileUploadRepository.scala

```diff
@@ -32,6 +32,8 @@ class FileUploadRepository @Inject()(mongo: MongoComponent)(implicit ec: Executi
       collectionName = "FileUpload",
       domainFormat = FileUploadResponse.ufrFormat,
       indexes = Seq()) {
+  override lazy val requiresTtlIndex: Boolean = false
```

Comment:
The new Bootstrap version forces you to set up an index with TTL (Time To Live) on it - i.e. use mongo as a form of cache with most of the data being fed from downstream. The alternative is to set this flag to true. Otherwise an exception is thrown on creation. This is documented here: https://github.com/hmrc/hmrc-mongo/tree/main?tab=readme-ov-file#ttl-indexes

In CBC we're using Mongo as a permanent source of truth, thus I'm adding this flag so that we don't have to set up this index.